core/src/main/java/edu/ucr/cs/riple/core/Annotator.java

```diff
@@ -256,11 +257,24 @@ private void forceResolveRemainingErrors(
     // the method level.
     Set<AddAnnotation> nullUnMarkedAnnotations =
         remainingErrors.stream()
-            // filter non-method regions.
-            .filter(error -> !error.encMethod().equals("null"))
             // find the corresponding method nodes.
-            .map(error -> tree.findNode(error.encMethod(), error.encClass()))
-            // impossible, just sanity check or future nullness checker hints
+            .map(
+                error -> {
+                  if (!error.encMethod().equals("null")) {
+                    return tree.findNode(error.encMethod(), error.encClass());
+                  }
+                  if (error.nonnullTarget == null) {
+                    // Just a sanity check.
+                    return null;
+                  }
+                  // method called in initialization region of class and received null for
+                  // arguments.
```

Comment:
Suggested wording change, for clarity:
```suggestion
                  // For methods invoked in an initialization region, where the error is that `@Nullable` is being passed as an argument, we add a `@NullUnmarked` annotation to the called method.
```